addons/udes_stock/models/stock_move_line.py

```diff
@@ -775,3 +776,24 @@ def any_destination_locations_default(self):
         )
         default_dest.ensure_one()
         return any(ml.location_dest_id == default_dest for ml in self)
+
+    def package_name_policy(self):
+        """ Given a move line compute the next package name according to
+            the policy assigned to the its picking picking type.
+            If no policy is assigned the default policy is applied.
+        """
+        self.ensure_one()
+        policy = self.picking_id.picking_type_id.u_new_package_policy
+        name = None
+        if policy:
+            func = getattr(self, 'new_package_' + policy, None)
```

Comment:
May be sensible to have the constructed methods called "new_package_name_" + policy, since they return a name (rather than returning a package).